api/test/crossReference.test.js

```diff
@@ -0,0 +1,19 @@
+import fetch from "node-fetch";
+
+describe("cross reference", () => {
+  test("should return more than one component", async () => {
+    const res = await fetch(`${API_BASE}/external-db/HMR%202.0/HMR_1157`);
+
+    const { components } = await res.json();
+    expect(components.length).toBeGreaterThan(1);
+  });
+
+  test("should return a TSV file for a given data source", async () => {
```

Comment:
```suggestion
  test("should return a JSON object for a given data source", async () => {
```

Reply:
Thanks for spotting this! The description was actually pasted from another file (data overlay) and I forgot to change the description. I changed it to a more appropriate one here: https://github.com/MetabolicAtlas/MetabolicAtlas/pull/830/commits/fdafc41c6e54b1379ff62da43a20264cc3b869f2